### PR TITLE
Endurance 21 telemetered ingestions

### PR DESCRIPTION
**Pull request by cwingard**
@kpolitano One more for old times' sake!

### CODE REVIEW

**Review by kpolitano — COMMENTED** [review]
do we want to change the status of the 01 mopak to unavailable seeing as we already know it's not working?

File: CE06ISSM/CE06ISSM_D00020_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/presf/*.presf.log,CE06ISSM-MFD35-02-PRESFA000,telemetered,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/adcpt/*.adcpt.log,CE06ISSM-MFD35-04-ADCPTM000,telemetered,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/pco2w2/*.pco2w2.log,CE06ISSM-MFD35-05-PCO2WB000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/phsen2/*.phsen*.log,CE06ISSM-MFD35-06-PHSEND000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl37/syslog/*.syslog.log,CE06ISSM-MFD37-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl37/optaa2/*.optaa*.log,CE06ISSM-MFD37-01-OPTAAD000,telemetered,Available,
```
Comment (s-pearce) [inline CE06ISSM/CE06ISSM_D00020_ingest.csv:10]: No files found in the X00020 directory on the NAS drive.  Maybe these are "Not Expected" for status?

File: CE07SHSM/CE07SHSM_D00019_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl36/presf/*.presf.log,CE07SHSM-MFD35-02-PRESFB000,telemetered,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl36/adcpt*/*.adcpt*.log,CE07SHSM-MFD35-04-ADCPTC000,telemetered,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl36/pco2w/*.pco2w.log,CE07SHSM-MFD35-05-PCO2WB000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl36/phsen2/*.phsen*.log,CE07SHSM-MFD35-06-PHSEND000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl37/syslog/*.syslog.log,CE07SHSM-MFD37-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00019/cg_data/dcl37/optaa*/*.optaa*.log,CE07SHSM-MFD37-01-OPTAAD000,telemetered,Available,
```
Comment (s-pearce) [inline CE07SHSM/CE07SHSM_D00019_ingest.csv:10]: No \*optaa\*.log files found on X00019/cg_data/dcl37/optaa2_decimated/  should this be "Not Expected" in status?

File: CE09OSSM/CE09OSSM_D00019_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl36/presf/*.presf.log,CE09OSSM-MFD35-02-PRESFC000,telemetered,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl36/adcps/*.adcps.log,CE09OSSM-MFD35-04-ADCPSJ000,telemetered,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl36/pco2w/*.pco2w.log,CE09OSSM-MFD35-05-PCO2WB000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl36/phsen*/*.phsen*.log,CE09OSSM-MFD35-06-PHSEND000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl37/syslog/*.syslog.log,CE09OSSM-MFD37-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/D00019/cg_data/dcl37/optaa*/*.optaa*.log,CE09OSSM-MFD37-01-OPTAAC000,telemetered,Available,
```
Comment (s-pearce) [inline CE09OSSM/CE09OSSM_D00019_ingest.csv:10]: no X00019/cg_data/dcl37/optaa\* directory found at all on the NAS drive.  Should the status be "Not Expected" and the line commented out?

**Review by s-pearce — COMMENTED** [review]
@cwingard, see my in-line comments.  There are a few places where the ingest calls out optaa and says they are available, but when I look in the burn-in data X directories in several cases there are no \*optaa\*.log files and in one case an \*optaa\* directory does not exist.  So I am asking if A. these just aren't showing up in burn in data (X directories) but will show up in deployed directories (D when they actually show up) or B. are they not expected to produce data and the ingest line should be commented out and the status marked as "Not Expected"?
If A. I will mark as approved.
If B. I will request the changes.

File: CE06ISSM/CE06ISSM_D00020_ingest.csv (hunk 1 above)
Comment (cwingard) [reply CE06ISSM/CE06ISSM_D00020_ingest.csv:10]: Correct, the OPTAA's did not arrive in time for burn-in, so most of those directories are indeed empty. They are now installed and will be telemetering data during the deployment.

File: CE07SHSM/CE07SHSM_D00019_ingest.csv (hunk 2 above)
Comment (cwingard) [reply CE07SHSM/CE07SHSM_D00019_ingest.csv:10]: as noted above

File: CE09OSSM/CE09OSSM_D00019_ingest.csv (hunk 3 above)
Comment (cwingard) [reply CE09OSSM/CE09OSSM_D00019_ingest.csv:10]: as noted above

**Review by s-pearce — APPROVED** [review]

**Review by kpolitano — APPROVED** [review]

**Pull request merged**